core/cloudflow-operator/src/main/scala/cloudflow/operator/event/ConfigInputChangeEvent.scala

```diff
@@ -180,23 +185,33 @@ object ConfigInputChangeEvent {
 
   val TopicsConfigPath                                  = "cloudflow.topics"
   def streamletConfigPath(streamletName: String)        = s"cloudflow.streamlets.$streamletName"
```

Comment:
We changed the config path for the runner from `cloudflow.runner.streamlets` to `cloudflow.runner.streamlet`.
https://github.com/lightbend/cloudflow/blob/master/core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletDefinition.scala#L118
Do we need to do this on this path as well?

Reply:
No this is the user -level config file